### PR TITLE
[MAISTRA-2067] [MAISTRA-1528] Fix bookinfo versions to pick up latest version and CVE fixes

### PR DESCRIPTION
**Pull request by brian-avery**
Please provide a description for what this PR is for.
This updates our Bookinfo examples to fix several CVEs. This also fixes tests due to the fact that Helm charts were moved as well as disables reliance in gcloud in tests.

### CODE REVIEW

File: prow/lib.sh [hunk 1]
```diff
@@ -14,6 +14,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+function setup_gcloud_credentials() {
```
Comment (brian-avery) [inline prow/lib.sh:17]: This pulls https://github.com/maistra/istio/commit/ed5002c1499766aa9a7bee842b4045cdeb125618#diff-09361e4e1d0f5c6062a58779c9b2b6aa2a959d7364762158cde572cbfe816808 in so that prow tests will run

**Comment by brian-avery** [comment]
Added a hold until after tag of 1.1.11

File: samples/bookinfo/platform/kube/bookinfo.yaml [hunk 2]
```diff
@@ -291,7 +291,7 @@ spec:
       serviceAccountName: bookinfo-productpage
       containers:
       - name: productpage
-        image: maistra/examples-bookinfo-productpage-v1:1.1.1
+        image: maistra/examples-bookinfo-productpage-v1:1.1.12
```
Comment (dgn) [inline samples/bookinfo/platform/kube/bookinfo.yaml:294]: did you push these tags already?

File: Makefile.core.mk [hunk 3]
```diff
@@ -634,7 +634,7 @@ $(HELM): $(ISTIO_OUT)
 	bin/init_helm.sh
 
 $(HOME)/.helm:
-	$(HELM) init --client-only
+	$(HELM) init --stable-repo-url https://charts.helm.sh/stable --client-only
```
Comment (dgn) [inline Makefile.core.mk:637]: was this change necessary to run locally or on prow?

File: prow/lib.sh (hunk 1 above)
Comment (dgn) [reply prow/lib.sh:17]: was this change necessary to run locally or on prow?

File: samples/bookinfo/platform/kube/bookinfo.yaml (hunk 2 above)
Comment (brian-avery) [reply samples/bookinfo/platform/kube/bookinfo.yaml:294]: I did. And tested to make sure that I can deploy them using the YAML files in this PR.

File: Makefile.core.mk (hunk 3 above)
Comment (brian-avery) [reply Makefile.core.mk:637]: It was necessary to run either locally or on prow. Helm killed the built-in default URL for charts after they ended support for Helm 2. This is an archive URL that Helm now suggests people use.

File: prow/lib.sh (hunk 1 above)
Comment (brian-avery) [reply prow/lib.sh:17]: It was necessary to run on prow.

File: Makefile.core.mk (hunk 3 above)
Comment (brian-avery) [reply Makefile.core.mk:637]: This is the error that was seen in prow:

`test-317de0adb52549ea/_suite_context/istio-deployment-173388722/helm/repository/repositories.yaml \nAdding stable repo with URL: https://kubernetes-charts.storage.googleapis.com \nError: error initializing: Looks like \"https://kubernetes-charts.storage.googleapis.com\" is not a valid chart repository or cannot be reached: Failed to fetch https://kubernetes-charts.storage.googleapis.com/index.yaml : 403 Forbidden\n"`

**Review by jwendell — APPROVED** [review]

File: Makefile.core.mk (hunk 3 above)
Comment (dgn) [reply Makefile.core.mk:637]: ah, thanks. so it's not related to a change in version on our side, but they killed the server. gotcha

**Review by dgn — APPROVED** [review]
lgtm, thanks for fixing the `gcloud` issue. did we establish why this is now broken on the 1.1 image?

**Comment by jwendell** [comment]
> lgtm, thanks for fixing the `gcloud` issue. did we establish why this is now broken on the 1.1 image?

My guess is that we rebuilt 1.1 for the last time (before removing the 1.1 dockerfiles) *without* the script you added with the fake gcloud.
